packages/blockchain/src/consensus/clique.ts

```diff
@@ -304,7 +304,7 @@ export class CliqueConsensus implements Consensus {
           activeSigners.push(beneficiary)
           activeSigners.sort((a, b) => {
             // Sort by buffer size
-            return a.toBuffer().compare(b.toBuffer())
+            return a.toBytes().compare(b.toBytes())
```

Comment:
The `compare` method does not existing on Uin8Arrays. Not sure what the best way of comparing Uint8Arrays is. Perhaps converting to BigInt and comparing those?